src/managers/KeybindManager.cpp

```diff
@@ -2148,6 +2149,40 @@ SDispatchResult CKeybindManager::resizeActive(std::string args) {
     return {};
 }
 
+SDispatchResult CKeybindManager::scaleActive(std::string args) {
+    const auto PLASTWINDOW = g_pCompositor->m_pLastWindow.lock();
+
+    if (!PLASTWINDOW)
+        return {.success = false, .error = "Window not found"};
+
+    std::optional<float> scaleResult;
+    bool                 exact = false;
+
+    if (args.starts_with("exact")) {
+        exact       = true;
+        scaleResult = getPlusMinusKeywordResult(args.substr(5), 0);
+    } else
+        scaleResult = getPlusMinusKeywordResult(args, 0);
+
+    if (!scaleResult.has_value()) {
+        Debug::log(ERR, "Invalid arg \"{}\" in scaleactive!", args);
+        return {.success = false, .error = "Invalid scale argument in scaleactive!"};
+    }
+
+    float scale = scaleResult.value();
+    if (!exact)
+        scale += 1.0 / PLASTWINDOW->m_sWindowData.contentScale.valueOr(1.0f);
+
+    if (scale > 0.0f)
```

Comment:
do we want to allow scale to be 0.001? I'd say no, because that might crash with the size of the buffer. I'd cap all scales at 0.2 or 0.5

Reply:
OK, I will cap it to 0.25 for example to create a clean x4 multiplier.